Classes/Parser/ScssParser.php

```diff
@@ -96,7 +96,7 @@ protected function parseFile($file, $settings)
             // Resolve potential back paths manually using PathUtility::getCanonicalPath,
             // but make sure we do not break out of TYPO3 application path using GeneralUtility::getFileAbsFileName
             // Also resolve EXT: paths if given
-            $isTypo3Absolute = strpos($url, 'EXT:') === 0;
+            $isTypo3Absolute = (strpos($url, 'EXT:') === 0) || (strpos($url, '/') === 0);
```

Comment:
be careful, absolute paths don't start with `/` on Windows.

Reply:
Yeah, you're right. Thanks for the hint!